eclass/coreos-doc.eclass

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+# @ECLASS: coreos-doc.eclass
+# @BLURB: utility function for including documentation
+
+# @FUNCTION: dodoc-coreos
+# @USAGE: [-r] <list of docs>
+dodoc-coreos() {
```

Comment:
nit but coreos as a prefix would be more inline with typical namespacing than a suffix.